airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_payment_methods.py

```diff
@@ -194,26 +246,42 @@ def test_given_http_status_500_once_before_200_when_read_then_retry_and_return_r
     @HttpMocker()
     def test_given_http_status_500_when_read_then_raise_config_error(self, http_mocker: HttpMocker) -> None:
```

Comment:
Outside of the scope of this PR: I know this test was already there (I probably wrote these when adding a safety net initially before adding concurrency) but the naming is very confusing. Basically, we have a system_error (because of the actual 500) and a config_error (in order to fail the stream without the platform sentry alert to sentry. I think the former is more important